---
title: A comprehensive proteomic and bioinformatics analysis of human spinal cord
  injury plasma identifies proteins associated with the complement cascade and liver
  function as potential prognostic indicators of neurological outcome
author: Mateus Harrington
date: '2022-10-25'
slug: []
categories:
  - Publications
tags:
  - Academic
  - Paper
  - Prognostic model
  - Proteomics
  - Spinal Cord Injury
excerpt: A comparison of the human plasma proteome between "imporvers" and "non-improvers" in spinal cord injury
featured: true
#layout: single-sidebar
links:
- icon: newspaper
  icon_pack: fas
  name: DOI 
  url: https://doi.org/10.1101/2022.07.12.499696
- icon: file-pdf
  icon_pack: fa
  name: PDF
  url: https://www.biorxiv.org/content/10.1101/2022.07.12.499696v3.full.pdf
# - icon: quotes
#   icon_pack: fa
#   name: Cite
#   url: cite.bib
subtitle: ""
layout: single
---

# Abstract

## Introduction

Spinal Cord Injury (SCI) is a major cause of disability, with complications post-injury often leading to life-long health issues with need of extensive treatment. 
Neurological outcome post-SCI can be variable and difficult to predict, particularly in incomplete injured patients. 
The identification of specific SCI biomarkers in blood, may be able to improve prognostics in the field. 
This study has utilised proteomic and bioinformatics methodologies to investigate differentially expressed proteins in plasma samples across human SCI cohorts with the aim of identifying prognostic biomarkers and biological pathway alterations that relate to neurological outcome.

## Methods and Materials

Blood samples were taken, following informed consent, from ASIA impairment scale (AIS) grade C “Improvers” (those who experienced an AIS grade improvement) and “Non-Improvers” (No AIS change), and AIS grade A and D at <2 weeks (“Acute”) and approx. 
3 months (“Sub-acute”) post-injury. 
The total protein concentration from each sample was extracted, with pooled samples being labelled and non-pooled samples treated with ProteoMiner™ beads. 
Samples were then analysed using two 4-plex isobaric tag for relative and absolute quantification (iTRAQ) analyses and a label-free experiment for comparison, before quantifying with mass spectrometry. 
Data are available via ProteomeXchange with identifiers PXD035025 and PXD035072 for the iTRAQ and label-free experiments respectively.

Proteomic datasets were analysed using OpenMS (version 2.6.0). R (version 4.1.4) and in particular, the R packages MSstats (version 4.0.1) and pathview (version 1.32.0) were used for downstream analysis. 
Proteins of interest identified from this analysis were further validated by enzyme-linked immunosorbent assay (ELISA).

## Results

The data demonstrated proteomic differences between the cohorts, with the results from the iTRAQ approach supporting those of the label-free analysis. 
A total of 79 and 87 differentially abundant proteins across AIS and longitudinal groups were identified from the iTRAQ and label-free analyses, respectively. 
Alpha-2-macroglobulin (A2M), retinol binding protein 4 (RBP4), serum amyloid A1 (SAA1), Peroxiredoxin 2, alipoprotein A1 (ApoA1) and several immunoglobulins were identified as biologically relevant and differentially abundant, with potential as individual prognostic biomarkers of neurological outcome. 
Bioinformatics analyses revealed that the majority of differentially abundant proteins were components of the complement cascade and most interacted directly with the liver.

## Conclusions

Many of the proteins of interest identified using proteomics were detected only in a single group and therefore have potential as a binary (present or absent) biomarkers, RBP4 and PRX-2 in particular. 
Additional investigations into the chronology of these proteins, and their levels in other tissues (cerebrospinal fluid in particular) are needed to better understand the underlying pathophysiology, including any potentially modifiable targets. 
Pathway analysis highlighted the complement cascasde as being significant across groups of differential functional recovery.